core/src/main/scala/frameless/CatalystBround.scala

```diff
@@ -0,0 +1,19 @@
+package frameless
+
+import scala.annotation.implicitNotFound
+
+/** Spark does not return always the same type on abs as the input was
+  */
+@implicitNotFound("Cannot compute bround on type ${In}.")
+trait CatalystBround[In, Out]
+
+object CatalystBround {
+  private[this] val theInstance = new CatalystBround[Any, Any] {}
+  private[this] def of[In, Out]: CatalystBround[In, Out] = theInstance.asInstanceOf[CatalystBround[In, Out]]
+
+  implicit val framelessAbsoluteBigDecimal: CatalystBround[BigDecimal, java.math.BigDecimal]  = of[BigDecimal, java.math.BigDecimal]
```

Comment:
It looks like these instance are exactly the same as the ones in `CatalystAbsolute`. Instead of adding a new type function, do you think you could rename `CatalystAbsolute` with a more generic name, and use it for both `abs` and `bround`?

Reply:
ok

Reply:
ok